linux_os/guide/system/bootloader-grub2/grub2_uefi_admin_username/rule.yml

```diff
@@ -0,0 +1,77 @@
+documentation_complete: true
+
+prodtype: fedora,ocp4,ol7,ol8,rhel7,rhel8,rhv4,wrlinux1019
+
+title: 'Set the UEFI Boot Loader Admin Username to a Non-Default Value'
+
+description: |-
+    The grub2 boot loader should have a superuser account and password
+    protection enabled to protect boot-time settings.
+    <br /><br />
+    To maximize the protection, select a password-protected superuser account with unique name, and modify the
+    <tt>/etc/grub.d/01_users</tt> configuration file to reflect the account name change.
+    <br /><br />
+    Do not to use common administrator account names like root,
+    admin, or administrator for the grub2 superuser account.
```

Comment:
```suggestion
    It is highly suggested not to use common administrator account names like root,
    admin, or administrator for the grub2 superuser account.
```

Reply:
Unless we can point to something that explicitly forbids this, the language should be adjusted to reflect this is a suggestion.